api/search.js

```diff
@@ -6,18 +6,63 @@ const get = require('lodash/get');
 const auth = require('./auth');
 const User = require('./users/userModel');
 
-const router = express.Router();
-
 const esClient = new elasticsearch.Client({
   host: process.env.ELASTIC_SEARCH_HOST,
   log: 'trace',
   apiVersion: '7.2', // use the same version of your Elasticsearch instance
 });
 
+const router = express.Router();
+
 function isString(value) {
   return typeof value === 'string' || value instanceof String;
 }
 
+// eslint-disable-next-line no-unused-vars
```

Comment:
@yuvalbardugo Please STOP disabling every comment you have ! Solve the issue instead. We have a rule that if you have unused vars they should start with an underscore

Reply:
You do know this part of the code is not mine?

Reply:
This part of the code is not mine. I just left it there as it was.